### PR TITLE
link fixes and typos

### PR DESCRIPTION
**Pull request by hallaroo**

### CODE REVIEW

File: content/en/docs/faq/configuration.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ Yes. You can start vttablet with the `-init_db_name_override` command line optio
 
 ### How do I connect to vtgate using MySQL protocol?
 
-If you look at the example [vtgate-up.sh](https://github.com/vitessio/vitess/blob/master/examples/local/vtgate-up.sh) script, you'll see the following lines:
+If you look at the legacy example [vtgate-up.sh](https://github.com/vitessio/vitess/blob/master/examples/legacy_local/scripts/vtgate-up.sh) script, you'll see the following lines:
```
Comment (aquarapid) [inline content/en/docs/faq/configuration.md]: Don't think we need to refer to the legacy examples here;  the differences between legacy and non-legacy are not significant for the purposes of this doc.

File: content/en/docs/reference/features/sharding.md [hunk 2]
```diff
@@ -94,6 +94,6 @@ Cool a hot tablet | For read access, add replicas or split shards. For write acc
 
 Vitess provides the following tools to help manage range-based shards:
 
-* The [vtctl](../vtctl) command-line tool supports functions for managing keyspaces, shards, tablets, and more.
+* The [vtctl](../../vtctl) command-line tool supports functions for managing keyspaces, shards, tablets, and more.
 * Client APIs account for sharding operations.
-* The [MapReduce framework](https://github.com/vitessio/vitess/tree/master/java/hadoop/src/main/java/io/vitess/hadoop) fully utilizes key ranges to read data as quickly as possible, concurrently from all shards and all replicas.
+* The [MapReduce framework](https://github.com/vitessio/vitess/tree/d8764de783d61ca8a637b76a44d1cefbef75b935/java/hadoop/src/main/java/io/vitess/hadoop) fully utilizes key ranges to read data as quickly as possible, concurrently from all shards and all replicas.
```
Comment (aquarapid) [inline content/en/docs/reference/features/sharding.md]: This has been deprecated and removed, we can remove it from the docs.

File: content/en/docs/faq/configuration.md (hunk 1 above)
Comment (hallaroo) [reply content/en/docs/faq/configuration.md]: Fixed

File: content/en/docs/reference/features/sharding.md (hunk 2 above)
Comment (hallaroo) [reply content/en/docs/reference/features/sharding.md]: Removed

**Review by aquarapid — APPROVED** [review]

**Pull request merged**